edenia/eden-spend-explorer
# Fix/expense higher than income 433

### Expenses higher than income

### What does this PR do?

- Resolve #443

### Steps to test

1. run the project
2. got to delegate view and see the expenses and incomes in the right way
